block.go

```diff
@@ -16,8 +16,12 @@ type Disk struct {
 	SizeBytes              uint64
 	PhysicalBlockSizeBytes uint64
 	BusType                string
+	BusPath                string
+	NumaNodeId             int
```

Comment:
Please name the attribute "NUMANodeID". Per Golang naming convention best practices, acronyms should remain capitalized:

https://talks.golang.org/2014/names.slide#5

I need to go through `ghw` and double-check that all the field names follow these conventions...

Reply:
Noted. Thanks!